content/docs/querying/api.md

```diff
@@ -337,3 +337,73 @@ String results are returned as result type `string`. The corresponding
 ```
 [ <unix_time>, "<string_value>" ]
 ```
+
+
+## Targets
+
+> This API is in beta as it is intended to be extended with targets dropped due
```

Comment:
Actually we used `experimental` and `beta` in the past depending on the stability degree. Experimental sounds good though for the very first release.